content/source/docs/cloud/overview.html.md

```diff
@@ -0,0 +1,147 @@
+
+- When new commits are merged, affected workspaces automatically run Terraform plans with the new code.
+- When pull requests are opened, affected workspaces run speculative plans with the proposed code changes and post the results as a pull request check; reviewers can see at a glance whether the plan was successful, and can click through to view the proposed changes in detail.
+
+VCS integration is powerful, but optional; if you use an unsupported VCS or want to preserve an existing validation and deployment pipeline, you can use the API or Terraform CLI to upload new configuration versions. You'll still get the benefits of remote execution and Terraform Cloud's other features.
+
+- More info: [VCS-driven Runs](./run/ui.html)
+- More info: [Supported VCS Providers](./vcs/index.html#supported-vcs-providers)
+
+### Command Line Integration
+
+Remote execution offers major benefits to a team, but local execution offers major benefits to individual developers; for example, most Terraform users run `terraform plan` to interactively check their work while editing configurations.
+
+Terraform Cloud offers the best of both worlds, allowing you to run remote plans from your local command line. Configure the [remote backend][], and the `terraform plan` command will start a remote run in the configured Terraform Cloud workspace. The output of the run streams directly to your terminal, and you can also share a link to the remote run with your teammates.
+
+Remote CLI-driven runs use the current working directory's Terraform configuration and the remote workspace's variables, so you don't need to obtain production cloud credentials just to preview a configuration change.
+
+The remote backend also supports state manipulation commands like `terraform import` or `terraform taint`.
+
+-> **Note:** When used with Terraform Cloud, the `terraform plan` command runs [speculative plans][], which preview changes without modifying real infrastructure. You can also use `terraform apply` to perform full remote runs, but only with workspaces that are _not_ connected to a VCS repository. This helps ensure that your VCS remains the source of record for all real infrastructure changes.
+
+- More info: [CLI-driven Runs](./run/cli.html)
+
+### Cross-Workspace Data Sharing
+
+Infrastructure is never completely isolated, and separate groups of infrastructure resources often need to share a small amount of information.
+
+Terraform Cloud makes it trivial to share information between workspaces with root-level [outputs][]. Any workspace that uses remote operations can use [`terraform_remote_state` data sources][remote_state] to access other workspaces' outputs, without any additional configuration or authentication.
```

Comment:
Actually, I would put `Run Triggers` as its own section -- could we move this to the top and rename it `Remote State in Terraform Cloud` (right before `Remote Terraform Execution`)?

The introduction for Terraform Workflow reads: 

> At minimum, state must be shared; ideally, Terraform should execute in a consistent remote environment.

So it might be better to make remote state one of the first "features" for TFC.

Reply:
Hmm. Need to think about this a bit more.

Reply:
Yeah, I think this is a decent suggestion. Though it belongs more in the middle than the beginning; since state storage is tied to workspaces, it should come after that.